proto/zetachain/zetacore/observer/params.proto

```diff
@@ -34,6 +34,10 @@ message ChainParams {
 
   // Advanced confirmation parameters for chain to support fast observation
   ConfirmationParams confirmation_params = 18;
+
+  // Skip actions that require scanning the contents of each block.
+  // The main thing this disables is transfers directly to the TSS address.
+  bool skip_block_scan = 19;
```

Comment:
Wouldn't disable better wording than skip?

Skip sounds like skipping an optional step in a process for me, while this disable a feature by itself

Reply:
Updated